doc/manipulating_images/masker_objects.rst

```diff
@@ -101,7 +101,7 @@ slice and create a :ref:`Niimg <niimg>` in memory:
 
 
 .. literalinclude:: ../../examples/06_manipulating_images/plot_mask_computation.py
-    :start-after: Load movie watching based brain development fmri dataset
+    :start-after: Load movie watching based brain development fMRI dataset
```

Comment:
Why not :term:`fMRI` ?

Reply:
because coding late leads to mistakes...

Reply:
I'm not sure because if it doesn't apply to the example, changing it here will fail the reference. It is a comment inside a code block in the referenced example so I don't think the term role should apply